### PR TITLE
docs: Add migrate docker compose guide

### PR DESCRIPTION
**Pull request by sixhobbits**
**What this PR does / why we need it**:

Adds a guide to help people migrate from Docker Compose to Garden

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/guides/migrating-from-docker-compose.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+![To Do](./img/todo.png)
+
+## Larger migrations
+
+This was a simple example, but it should give you what you need to migrate larger projects too. If you have feedback on how we could make migrating from Docker Compose easier, please send us feedback via [GitHub issues](https://github.com/garden-io/garden/issues), or reach out on our Slack channel.
+
+
+
+
+
```
Comment (thelongshanx) [inline docs/guides/migrating-from-docker-compose.md:168]: remove trailing lines

**Comment by thelongshanx** [comment]
Thank you so much - this works & looks great. We'll get the checks fixed, and if you could pull in those suggested edits, we'll get it merged!

**Review by thelongshanx — DISMISSED** [review]

**Comment by sixhobbits** [comment]
@thelongshanx - thanks for the feedback - changes done.

**Review by eysi09 — APPROVED** [review]

**Comment by Orzelius** [comment]
Heyy, next time could you do a docs render after an update by running `yarn generate-docs` in the repo root please

**Pull request merged**